src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -70,7 +70,7 @@
 @RequestMapping(path = "/",
     consumes = MediaType.APPLICATION_JSON_VALUE,
     produces = MediaType.APPLICATION_JSON_VALUE)
-@Api(value = "/", description = "Standard case API")
```

Comment:
This will impact the Swagger docs, do we need to delete this?

Reply:
Description , parameter.  is deprecated  - advised by sonar to remove

Reply:
Can we replace with the non-deprecated then, rather than deleting. See `CaseSearchEndpoint` for example.

Reply:
Updated